slack-notification/send-slack-notification.ps1

```diff
@@ -0,0 +1,63 @@
+<#
+.SYNOPSIS
+Sending messages using Incoming Webhooks
+
+.PARAMETER Url
+Required parameter. Incoming Webhook URL to post a message
+.PARAMETER ToolName
+Required parameter. The name of tool
+.PARAMETER ToolVersion
+Required parameter. Specifies the version of tool
+.PARAMETER PipelineUrl
+Required parameter. The pipeline URL
+.PARAMETER ImageUrl
+Optional parameter. The image URL
+#>
+
+param(
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [System.Uri]$Url,
+
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [System.String]$ToolName,
+
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [System.String]$ToolVersion,
+
+    [Parameter(Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [System.String]$PipelineUrl,
+
+    [System.String]$ImageUrl = 'https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png'
+)
+
+# Import helpers module
+Import-Module $PSScriptRoot/slack.helpers.psm1 -DisableNameChecking
+
+# Create JSON body
+$text = "The following versions of '$toolName' are available to upload: $toolVersion\nLink to the pipeline: $pipelineUrl"
+$jsonBodyMessage = @"
+{
+    "blocks": [
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": "$text"
+            },
+            "accessory": {
+                "type": "image",
+                "image_url": "$imageUrl",
+                "alt_text": "$toolName"
+            }
+        }
+    ]
+}
+"@
+
+# Send Slack message
+$null = Send-SlackPostMessageIncomingWebHook -Uri $url -Body $jsonBodyMessage
```

Comment:
May be we should put this script to `get-new-tool-versions` folder. Near `get-new-tool-versions/get-new-tool-versions.ps1` because looks like this script is intended only for pushing slack notifications about new versions from runner.
We may still keep `slack.helpers.psm1` in the `slack-notification` folder

Reply:
Done.